Source/Core/Logging/Theme.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Theme.swift
+//  Frames
+//
+//  Created by Harry Brown on 23/03/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import CheckoutEventLoggerKit
+
+class Theme: PropertyProviding, Equatable {
```

Comment:
```suggestion
final class Theme: PropertyProviding, Equatable {
```

Reply:
if we change this one then `StubTheme` can't extend from `Theme`. This in turns means that we need to make a custom equals function on `FramesLogEvent`.

Not sure which is preferable